src/dftbp/derivs/linearresponse.F90

```diff
@@ -80,10 +80,10 @@ subroutine dRhoReal(env, dHam, neighbourList, nNeighbourSK, iSparseStart, img2Ce
     !> Particular spin/k-point
     integer, intent(in) :: iKS
 
-    !> K-points and spins to process
+    !> k-points and spins to process
```

Comment:
Interesting question, as this is usually a lower case mathematical variable, and does look a bit weird at the start of a line.
```suggestion
    !> The k-points and spins to process
```

Reply:
True, thanks for the suggestion, I will just apply it. Of course this particular case occurs all over the codebase, so let's remember this convention and successively apply it.